reminders/class_tests/create_reminder_tests.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+from datetime import datetime
+from django.utils import timezone
+from reminders.models import ReminderType
+from reminders.forms import ReminderCreationForm
+from events.models import EventParticipant, Colors
+from pytest_django.asserts import assertTemplateUsed
+
+
+NONE_METHOD = None
+INVALID_METHOD = "undefined"
+CURRENT_DATE = datetime.now()
+VALID_METHOD = ReminderType.EMAIL
+MESSAGE = "Joined Meeting in 15 minutes"
+VALID_DATE_TIME = datetime(CURRENT_DATE.year, CURRENT_DATE.month, CURRENT_DATE.day,
+                           12, 0, 0, tzinfo=timezone.utc)
+INVALID_DATE_TIME = datetime(CURRENT_DATE.year - 1, CURRENT_DATE.month, CURRENT_DATE.day,
+                             12, 0, 0, tzinfo=timezone.utc)
+
+DATE_TIME_END = datetime(CURRENT_DATE.year, CURRENT_DATE.month, CURRENT_DATE.day,
+                         13, 0, 0, tzinfo=timezone.utc)
+DATE_TIME_START = datetime(CURRENT_DATE.year, CURRENT_DATE.month, CURRENT_DATE.day,
+                           15, 0, 0, tzinfo=timezone.utc)
+REMINDER_DATE_TIME = datetime(CURRENT_DATE.year, CURRENT_DATE.month, CURRENT_DATE.day,
+                              14, 0, 0, tzinfo=timezone.utc)
+
+
+REMINDER_CREATION_URL = '/event/create/'
+REMINDER_CREATION_HTML_PATH = "events/create_event.html"
+
+
+@pytest.mark.django_db
+class TestCreateReminder:
+
+    @pytest.fixture
+    def signed_up_user_details(self):
+        return {'email': 'admin@gmail.com', 'password': 'adminPassword'}
+
+    @pytest.fixture
+    def sign_in(self, client, signed_up_user_details):
+        return client.post('/login/', data=signed_up_user_details)
+
+    @pytest.fixture
+    def get_event_participant(self):
+        return EventParticipant.objects.get(id=1)
+
+    @pytest.fixture
+    def valid_event_data(self):
+        return {'title': "test event", 'date_time_start': DATE_TIME_START,
+                'date_time_end': DATE_TIME_END, 'color': Colors.BLACK,
+                'date_time': REMINDER_DATE_TIME, 'method': ReminderType.EMAIL}
```

Comment:
In my opinion you can use conftest.py file (which providing constants and fixtuers to entire directory), for the reminder app and put there all the constants and fixtures. In this way, you can use all the constants and fixtures in each test file in the app without importing them. 
I think that it can make our code more readable and maintainable 😄 

Here is link to conftest.py documentation:
https://docs.pytest.org/en/6.2.x/fixture.html#conftest-py-sharing-fixtures-across-multiple-files

Reply:
Great Catch, 💪 
I will add a new file conftest.py to the app directory so 
that all the tests will have a global area to share fixtures and data. 

done. 😄

Reply:
Great! 😄